Description: Fix all spotted hyphen-used-as-minus-sign manpage errors
 Automatically generated using sed -i -s 's/^-/\\-/g;s/ -/ \\-/g' man/*.man*
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Bug-Upstream: https://cups.org/str.php?L4299
Last-Update: 2013-03-20

--- a/man/cups-files.conf.man.in
+++ b/man/cups-files.conf.man.in
@@ -53,7 +53,7 @@
 .TP 5
 FatalErrors none
 .TP 5
-FatalErrors all -kind [... -kind]
+FatalErrors all \-kind [... \-kind]
 .TP 5
 FatalErrors kind [... kind]
 .br
--- a/man/cupsd.man.in
+++ b/man/cupsd.man.in
@@ -17,9 +17,9 @@
 cupsd \- cups scheduler
 .SH SYNOPSIS
 .B cupsd
-[ -c
+[ \-c
 .I config-file
-] [ -f ] [ -F ] [ -h ] [ -l ] [ -t ]
+] [ \-f ] [ \-F ] [ \-h ] [ \-l ] [ \-t ]
 .SH DESCRIPTION
 \fIcupsd\fR is the scheduler for CUPS. It implements a printing system based
 upon the Internet Printing Protocol, version 2.1.  If no options are specified
@@ -27,31 +27,31 @@
 \fI@CUPS_SERVERROOT@/cupsd.conf\fR will be used.
 .SH OPTIONS
 .TP 5
--c config-file
+\-c config-file
 .br
 Uses the named configuration file.
 .TP 5
--f
+\-f
 .br
 Run \fIcupsd\fR in the foreground; the default is to run in the
 background as a "daemon".
 .TP 5
--F
+\-F
 .br
 Run \fIcupsd\fR in the foreground but detach the process from the
 controlling terminal and current directory. This is useful for
 running \fIcupsd\fR from \fIinit(8)\fR.
 .TP 5
--h
+\-h
 .br
 Shows the program usage.
 .TP 5
--l
+\-l
 .br
 This option is passed to \fIcupsd\fR when it is run from
 \fIlaunchd(8)\fR.
 .TP 5
--t
+\-t
 .br
 Test the configuration file for syntax errors.
 .SH COMPATIBILITY
--- a/man/cupsfilter.man
+++ b/man/cupsfilter.man
@@ -16,27 +16,27 @@
 cupsfilter \- convert a file to another format using cups filters
 .SH SYNOPSIS
 .B cupsfilter
-[ -D ] [ -U
+[ \-D ] [ \-U
 .I user
-] [ -c
+] [ \-c
 .I config-file
-] [ -d
+] [ \-d
 .I printer
-] [ -e ] [ -i
+] [ \-e ] [ \-i
 .I mime/type
-] [ -j
+] [ \-j
 .I job-id[,N]
-] [ -m
+] [ \-m
 .I mime/type
-] [ -n
+] [ \-n
 .I copies
-] [ -o
+] [ \-o
 .I name=value
-] [ -p
+] [ \-p
 .I filename.ppd
-] [ -t
+] [ \-t
 .I title
-] [ -u ]
+] [ \-u ]
 .I filename
 .SH DESCRIPTION
 \fIcupsfilter\fR is a front-end to the CUPS filter subsystem which allows you
@@ -44,58 +44,58 @@
 through CUPS. By default, \fIcupsfilter\fR generates a PDF file.
 .SH OPTIONS
 .TP 5
--D
+\-D
 .br
 Delete the input file after conversion.
 .TP 5
--U user
+\-U user
 .br
 Specifies the username passed to the filters. The default is the name of the current user.
 .TP 5
--c config-file
+\-c config-file
 .br
 Uses the named cupsd.conf configuration file.
 .TP 5
--d printer
+\-d printer
 Uses information from the named printer.
 .TP 5
--e
+\-e
 .br
 Use every filter from the PPD file.
 .TP 5
--i mime/type
+\-i mime/type
 .br
 Specifies the source file type. The default file type is guessed using the
 filename and contents of the file.
 .TP 5
--j job-id[,N]
+\-j job-id[,N]
 .br
 Converts document N from the specified job. If N is omitted, document 1 is
 converted.
 .TP 5
--m mime/type
+\-m mime/type
 .br
 Specifies the destination file type. The default file type is application/pdf.
 Use printer/foo to convert to the printer format defined by the filters in the
 PPD file.
 .TP 5
--n copies
+\-n copies
 .br
 Specifies the number of copies to generate.
 .TP 5
--o name=value
+\-o name=value
 .br
 Specifies options to pass to the CUPS filters.
 .TP 5
--p filename.ppd
+\-p filename.ppd
 .br
 Specifies the PPD file to use.
 .TP 5
--t title
+\-t title
 .br
 Specifies the document title.
 .TP 5
--u
+\-u
 .br
 Delete the PPD file after conversion.
 .SH SEE ALSO
--- a/man/lpq.man
+++ b/man/lpq.man
@@ -17,13 +17,13 @@
 lpq \- show printer queue status
 .SH SYNOPSIS
 .B lpq
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ -P
+] [ \-P
 .I destination[/instance]
-] [ -a ] [ -l ] [
+] [ \-a ] [ \-l ] [
 .I +interval
 ]
 .SH DESCRIPTION
@@ -37,27 +37,27 @@
 .SH OPTIONS
 \fIlpq\fR supports the following options:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--P destination[/instance]
+\-P destination[/instance]
 .br
 Specifies an alternate printer or class name.
 .TP 5
--U username
+\-U username
 .br
 Specifies an alternate username.
 .TP 5
--a
+\-a
 .br
 Reports jobs on all printers.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Specifies an alternate server.
 .TP 5
--l
+\-l
 .br
 Requests a more verbose (long) reporting format.
 .SH SEE ALSO
--- a/man/backend.man
+++ b/man/backend.man
@@ -53,7 +53,7 @@
 Unless you are a developer and know what you are doing, please do not run
 backends directly. Instead, use the \fIlp(1)\fR or \fIlpr(1)\fR programs to send
 a print job or \fIlpinfo(8)\fR program to query for available printers using the
-backend. The one exception is the SNMP backend - see \fIsnmpbackend(8)\fR for
+backend. The one exception is the SNMP backend \- see \fIsnmpbackend(8)\fR for
 more information.
 .SH DEVICE DISCOVERY
 When run with no arguments, the backend should list the devices and schemes it
@@ -151,7 +151,7 @@
 6 (CUPS_BACKEND_RETRY)
 .br
 The print file was not successfully transmitted because of a temporary issue.
-The scheduler will retry the job at a future time - other jobs may print before
+The scheduler will retry the job at a future time \- other jobs may print before
 this one.
 .TP 5
 7 (CUPS_BACKEND_RETRY_CURRENT)
--- a/man/cancel.man
+++ b/man/cancel.man
@@ -14,14 +14,14 @@
 .\"
 .TH cancel 1 "CUPS" "20 March 2006" "Apple Inc."
 .SH NAME
-cancel - cancel jobs
+cancel \- cancel jobs
 .SH SYNOPSIS
 .B cancel
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -a ] [ -h
+] [ \-a ] [ \-h
 .I hostname[:port]
-] [ -u
+] [ \-u
 .I username
 ] [
 .I id
@@ -36,24 +36,24 @@
 .SH OPTIONS
 The following options are recognized by \fIcancel\fR:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--U username
+\-U username
 .br
 Specifies the username to use when connecting to the server.
 .TP 5
--a
+\-a
 .br
 Cancel all jobs on the named destination, or all jobs on all
 destinations if none is provided.
 .TP 5
--h hostname[:port]
+\-h hostname[:port]
 .br
 Chooses an alternate server.
 .TP 5
--u username
+\-u username
 .br
 Cancels jobs owned by \fIusername\fR.
 .SH COMPATIBILITY
--- a/man/cups-config.man
+++ b/man/cups-config.man
@@ -17,38 +17,38 @@
 cups-config \- get cups api, compiler, directory, and link information.
 .SH SYNOPSIS
 .B cups-config
---api-version
+\--api-version
 .br
 .B cups-config
---build
+\--build
 .br
 .B cups-config
---cflags
+\--cflags
 .br
 .B cups-config
---datadir
+\--datadir
 .br
 .B cups-config
---help
+\--help
 .br
 .B cups-config
---ldflags
+\--ldflags
 .br
 .B cups-config
 [
-.I --image
+.I \--image
 ] [
-.I --static
-] --libs
+.I \--static
+] \--libs
 .br
 .B cups-config
---serverbin
+\--serverbin
 .br
 .B cups-config
---serverroot
+\--serverroot
 .br
 .B cups-config
---version
+\--version
 .br
 .SH DESCRIPTION
 \fBcups-config\fR is the CUPS program configuration utility. It should be
@@ -57,54 +57,54 @@
 directories for filters, configuration files, and drivers.
 .SH OPTIONS
 .TP 5
---api-version
+\--api-version
 .br
 Displays the current API version (major.minor).
 .TP 5
---build
+\--build
 .br
 Displays a system-specific build number.
 .TP 5
---cflags
+\--cflags
 .br
 Displays the necessary compiler options.
 .TP 5
---datadir
+\--datadir
 .br
 Displays the default CUPS data directory.
 .TP 5
---help
+\--help
 .br
 Displays the program usage message.
 .TP 5
---image
+\--image
 .br
 When used with \fI--libs\fR, adds the CUPS imaging library to the
 list of displayed libraries.
 .TP 5
---ldflags
+\--ldflags
 .br
 Displays the necessary linker options.
 .TP 5
---libs
+\--libs
 .br
 Displays the necessary librarys to link to.
 .TP 5
---serverbin
+\--serverbin
 .br
 Displays the default CUPS binary directory,
 where filters and backends are stored.
 .TP 5
---serverroot
+\--serverroot
 .br
 Displays the default CUPS configuration file directory.
 .TP 5
---static
+\--static
 .br
 When used with \fI--libs\fR, shows the static libraries instead
 of the default (shared) libraries.
 .TP 5
---version
+\--version
 .br
 Displays the full version number of the CUPS installation
 (major.minor.patch).
--- a/man/cups-deviced.man.in
+++ b/man/cups-deviced.man.in
@@ -25,7 +25,7 @@
 \fICUPS-Get-Devices\fR request. The output format is an IPP
 response message. The \fIrequest-id\fR argument is the request ID
 from the original IPP request, typically 1. The \fIlimit\fR
-argument is the limit value from the original IPP request - 0
+argument is the limit value from the original IPP request \- 0
 means no limit. The \fIuser-id\fR argument is the
 requesting-user-name value from the original IPP request.
 Finally, the \fIoptions\fR argument is a space-delimited list of
--- a/man/cups-driverd.man.in
+++ b/man/cups-driverd.man.in
@@ -34,7 +34,7 @@
 format is an IPP response message. The \fIrequest_id\fR argument
 is the request ID from the original IPP request, typically 1. The
 \fIlimit\fR argument is the limit value from the original IPP
-request - 0 means no limit. Finally, the \fIoptions\fR argument
+request \- 0 means no limit. Finally, the \fIoptions\fR argument
 is a space-delimited list of attributes ("name=value name=value
 \&...") that were passed in with the request. Currently
 \fIcups-driverd\fR looks for the \fIppd-make\fR and
--- a/man/cups-lpd.man.in
+++ b/man/cups-lpd.man.in
@@ -17,9 +17,9 @@
 cups-lpd \- receive print jobs and report printer status to lpd clients
 .SH SYNOPSIS
 .B cups-lpd
-[ -h
+[ \-h
 .I hostname[:port]
-] [ -n ] [ -o
+] [ \-n ] [ \-o
 .I option=value
 ]
 .SH DESCRIPTION
@@ -34,7 +34,7 @@
 .nf
 
     printer stream tcp nowait lp @CUPS_SERVERBIN@/daemon/cups-lpd cups-lpd \\
-        -o document-format=application/octet-stream
+        \-o document-format=application/octet-stream
 .fi
 .LP
 .LP
@@ -56,22 +56,22 @@
 	group = sys
 	passenv =
         server = @CUPS_SERVERBIN@/daemon/cups-lpd
-	server_args = -o document-format=application/octet-stream
+	server_args = \-o document-format=application/octet-stream
     }
 .fi
 .SH OPTIONS
 .TP 5
--h hostname[:port]
+\-h hostname[:port]
 .br
 Sets the CUPS server (and port) to use.
 .TP 5
--n
+\-n
 .br
 Disables reverse address lookups; normally \fIcups-lpd\fR will
 try to discover the hostname of the client via a reverse DNS
 lookup.
 .TP 5
--o name=value
+\-o name=value
 .br
 Inserts options for all print queues. Most often this is used to
 disable the "l" filter so that remote print jobs are filtered as
--- a/man/cups-snmp.man.in
+++ b/man/cups-snmp.man.in
@@ -18,7 +18,7 @@
 .B @CUPS_SERVERBIN@/backend/snmp
 .I ip-address-or-hostname
 .br
-.B lpinfo -v --include-schemes snmp
+.B lpinfo \-v \--include-schemes snmp
 .SH DESCRIPTION
 The CUPS SNMP backend provides legacy discovery and identification of network
 printers using SNMPv1. When used for discovery through the scheduler, the
--- a/man/cupsaccept.man
+++ b/man/cupsaccept.man
@@ -17,18 +17,18 @@
 cupsaccept/cupsreject \- accept/reject jobs sent to a destination
 .SH SYNOPSIS
 .B cupsaccept
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I hostname[:port]
 ] destination(s)
 .br
 .B cupsreject
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I hostname[:port]
-] [ -r
+] [ \-r
 .I reason
 ] destination(s)
 .SH DESCRIPTION
@@ -42,19 +42,19 @@
 The following options are supported by both \fIcupsaccept\fR and
 \fIcupsreject\fR:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--U username
+\-U username
 .br
 Sets the username that is sent when connecting to the server.
 .TP 5
--h hostname[:port]
+\-h hostname[:port]
 .br
 Chooses an alternate server.
 .TP 5
--r "reason"
+\-r "reason"
 .br
 Sets the reason string that is shown for a printer that is
 rejecting jobs.
--- a/man/cupsaddsmb.man.in
+++ b/man/cupsaddsmb.man.in
@@ -18,22 +18,22 @@
 
 .SH SYNOPSIS
 .B cupsaddsmb
-[ -H
+[ \-H
 .I samba-server
-] [ -U
+] [ \-U
 .I samba-user[%samba-password]
-] [ -h
+] [ \-h
 .I cups-server[:port]
-] [ -v ] -a
+] [ \-v ] \-a
 .br
 .B cupsaddsmb
-[ -H
+[ \-H
 .I samba-server
-] [ -U
+] [ \-U
 .I samba-user[%samba-password]
-] [ -h
+] [ \-h
 .I cups-server[:port]
-] [ -v ] printer [ ... printer ]
+] [ \-v ] printer [ ... printer ]
 
 .SH DESCRIPTION
 \fIcupsaddsmb\fR exports printers to the SAMBA software (version
@@ -45,27 +45,27 @@
 .SH OPTIONS
 \fIcupsaddsmb\fR supports the following options:
 .TP 5
--H samba-server
+\-H samba-server
 .br
 Specifies the SAMBA server which defaults to the CUPS server.
 .TP 5
--U samba-user[%samba-password]
+\-U samba-user[%samba-password]
 .br
 Specifies the SAMBA print admin username which defaults to your
 current username. If the username contains a percent (%)
 character, then the text following the percent is treated as the
 SAMBA password to use.
 .TP 5
--a
+\-a
 .br
 Exports all known printers. Otherwise only the named printers are
 exported.
 .TP 5
--h cups-server[:port]
+\-h cups-server[:port]
 .br
 Specifies a different CUPS server to use.
 .TP 5
--v
+\-v
 .br
 Specifies that verbose information should be shown. This is
 useful for debugging SAMBA configuration problems.
@@ -136,7 +136,7 @@
     pscript5.dll
 .fi
 .LP
-\fBNote:\fR Unlike Windows, case is significant - make sure that
+\fBNote:\fR Unlike Windows, case is significant \- make sure that
 you use the lowercase filenames shown above, otherwise
 \fIcupsaddsmb\fR will fail to export the drivers.
 
@@ -169,7 +169,7 @@
     pscript5.dll (from your Windows system)
 .fi
 .LP
-\fBNote:\fR Unlike Windows, case is significant - make sure that
+\fBNote:\fR Unlike Windows, case is significant \- make sure that
 you use the lowercase filenames shown above, otherwise
 \fIcupsaddsmb\fR will fail to export the drivers.
 
@@ -194,7 +194,7 @@
     PSMON.DLL
 .fi
 .LP
-\fBNote:\fR Unlike Windows, case is significant - make sure that
+\fBNote:\fR Unlike Windows, case is significant \- make sure that
 you use the UPPERCASE filenames shown above, otherwise
 \fIcupsaddsmb\fR will fail to export the drivers.
 
--- a/man/cupsctl.man
+++ b/man/cupsctl.man
@@ -17,12 +17,12 @@
 cupsctl \- configure cupsd.conf options
 .SH SYNOPSIS
 .B cupsctl
-[ -E ] [-U
+[ \-E ] [\-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ --[no-]debug-logging ] [ --[no-]remote-admin ] [ --[no-]remote-any ]
-[ --[no-]share-printers ] [ --[no-]user-cancel-any ]
+] [ \-\-[no-]debug-logging ] [ \-\-[no-]remote-admin ] [ \-\-[no-]remote-any ]
+[ \-\-[no-]share-printers ] [ \-\-[no-]user-cancel-any ]
 [
 .I name=value
 ]
@@ -33,35 +33,35 @@
 .SH OPTIONS
 The following options are recognized:
 .TP 5
--E
+\-E
 .br
 Enables encryption on the connection to the scheduler.
 .TP 5
--U \fIusername\fR
+\-U \fIusername\fR
 .br
 Specifies an alternate username to use when authenticating with the scheduler.
 .TP 5
--h \fIserver[:port]\fR
+\-h \fIserver[:port]\fR
 .br
 Specifies the server address.
 .TP 5
---[no-]debug-logging
+\--[no-]debug-logging
 .br
 Enables or disables debug logging in the \fBerror_log\fR file.
 .TP 5
---[no-]remote-admin
+\--[no-]remote-admin
 .br
 Enables or disables remote administration.
 .TP 5
---[no-]remote-any
+\--[no-]remote-any
 .br
 Enables or disables printing from any address, e.g. the Internet.
 .TP 5
---[no-]share-printers
+\--[no-]share-printers
 .br
 Enables or disables sharing of local printers with other computers.
 .TP 5
---[no-]user-cancel-any
+\--[no-]user-cancel-any
 .br
 Allows or prevents users from canceling jobs owned by others.
 .SH EXAMPLES
@@ -72,17 +72,17 @@
 .LP
 Enable debug logging:
 .nf
-    cupsctl --debug-logging
+    cupsctl \--debug-logging
 .fi
 .LP
 Get the current debug logging state:
 .nf
-    cupsctl | grep '^_debug_logging' | awk -F= '{print $2}'
+    cupsctl | grep '^_debug_logging' | awk \-F= '{print $2}'
 .fi
 .LP
 Disable printer sharing:
 .nf
-    cupsctl --no-share-printers
+    cupsctl \--no-share-printers
 .fi
 .SH KNOWN ISSUES
 You cannot set the Listen or Port directives using \fIcupsctl\fR.
--- a/man/cupsenable.man
+++ b/man/cupsenable.man
@@ -17,53 +17,53 @@
 cupsdisable, cupsenable \- stop/start printers and classes
 .SH SYNOPSIS
 .B cupsdisable
-[ -E ] [-U
+[ \-E ] [\-U
 .I username
-] [ -c ] [ -h
+] [ \-c ] [ \-h
 .I server[:port]
-] [ -r
+] [ \-r
 .I reason
-] [ --hold ] destination(s)
+] [ \-\-hold ] destination(s)
 .br
 .B cupsenable
-[ -E ] [-U
+[ \-E ] [\-U
 .I username
-] [ -c ] [ -h
+] [ \-c ] [ \-h
 .I server[:port]
-] [ --release ] destination(s)
+] [ \--release ] destination(s)
 .SH DESCRIPTION
 \fIcupsenable\fR starts the named printers or classes.
 .LP
 \fIcupsdisable\fR stops the named printers or classes.  The
 following options may be used:
 .TP 5
--E
+\-E
 .br
 Forces encryption of the connection to the server.
 .TP 5
--U username
+\-U username
 .br
 Uses the specified username when connecting to the server.
 .TP 5
--c
+\-c
 .br
 Cancels all jobs on the named destination.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Uses the specified server and port.
 .TP 5
---hold
+\--hold
 .br
 Holds remaining jobs on the named printer.  Useful for allowing the current
 job to complete before performing maintenance.
 .TP 5
--r "reason"
+\-r "reason"
 .br
 Sets the message associated with the stopped state. If no reason is specified
 then the message is set to "Reason Unknown".
 .TP 5
---release
+\--release
 .br
 Releases pending jobs for printing. Use after running \fIcupsdisable\fR with
 the \fI--hold\fR option to resume printing.
--- a/man/cupstestdsc.man
+++ b/man/cupstestdsc.man
@@ -17,12 +17,12 @@
 cupstestdsc \- test conformance of postscript files
 .SH SYNOPSIS
 .B cupstestdsc
-[ -h ] filename.ps [ ...
+[ \-h ] filename.ps [ ...
 .I filenameN.ps
 ]
 .br
 .B cupstestdsc
-[ -h ] -
+[ \-h ] \-
 .SH DESCRIPTION
 \fIcupstestdsc\fR tests the conformance of PostScript files to
 the Adobe PostScript Language Document Structuring Conventions
--- a/man/cupstestppd.man
+++ b/man/cupstestppd.man
@@ -17,20 +17,20 @@
 cupstestppd \- test conformance of ppd files
 .SH SYNOPSIS
 .B cupstestppd
-[ -I
+[ \-I
 .I category
-] [ -R
+] [ \-R
 .I rootdir
-] [ -W
+] [ \-W
 .I category
-] [ -q ] [-r] [ -v[v] ] filename.ppd[.gz] [ ... filenameN.ppd[.gz] ]
+] [ \-q ] [\-r] [ \-v[v] ] filename.ppd[.gz] [ ... filenameN.ppd[.gz] ]
 .br
 .B cupstestppd
-[ -R
+[ \-R
 .I rootdir
-] [ -W
+] [ \-W
 .I category
-] [ -q ] [-r] [ -v[v] ] -
+] [ \-q ] [\-r] [ \-v[v] ] \-
 .SH DESCRIPTION
 \fIcupstestppd\fR tests the conformance of PPD files to the
 Adobe PostScript Printer Description file format specification
@@ -44,71 +44,71 @@
 .SH OPTIONS
 \fIcupstestppd\fR supports the following options:
 .TP 5
--I filename
+\-I filename
 .br
 Ignores all PCFileName warnings.
 .TP 5
--I filters
+\-I filters
 .br
 Ignores all filter errors.
 .TP 5
--I profiles
+\-I profiles
 .br
 Ignores all profile errors.
 .TP 5
--R rootdir
+\-R rootdir
 .br
 Specifies an alternate root directory for the filter, pre-filter,
 and other support file checks.
 .TP 5
--W constraints
+\-W constraints
 .br
 Report all UIConstraint errors as warnings.
 .TP 5
--W defaults
+\-W defaults
 .br
 Except for size-related options, report all default option errors as warnings.
 .TP 5
--W filters
+\-W filters
 .br
 Report all filter errors as warnings.
 .TP 5
--W profiles
+\-W profiles
 .br
 Report all profile errors as warnings.
 .TP 5
--W sizes
+\-W sizes
 .br
 Report all media size errors as warnings.
 .TP 5
--W translations
+\-W translations
 .br
 Report all translation errors as warnings.
 .TP 5
--W all
+\-W all
 .br
 Report all of the previous errors as warnings.
 .TP 5
--W none
+\-W none
 .br
 Report all of the previous errors as errors.
 .TP 5
--q
+\-q
 .br
 Specifies that no information should be displayed.
 .TP 5
--r
+\-r
 .br
 Relaxes the PPD conformance requirements so that common
 whitespace, control character, and formatting problems are not
 treated as hard errors.
 .TP 5
--v
+\-v
 .br
 Specifies that detailed conformance testing results should be
 displayed rather than the concise PASS/FAIL/ERROR status.
 .TP 5
--vv
+\-vv
 .br
 Specifies that all information in the PPD file should be
 displayed in addition to the detailed conformance testing
@@ -140,7 +140,7 @@
 conform:
 .nf
 
-    find . -name \\*.ppd \\! -exec cupstestppd -q '{}' \\; -print
+    find . \-name \\*.ppd \\! \-exec cupstestppd \-q '{}' \\; \-print
 
 .fi
 The next command tests all PPD files under the current directory
@@ -148,8 +148,8 @@
 that do not conform:
 .nf
 
-    find . -name \\*.ppd \\! -exec cupstestppd -q '{}' \\; \\
-        -exec cupstestppd -v '{}' \\;
+    find . \-name \\*.ppd \\! \-exec cupstestppd \-q '{}' \\; \\
+        \-exec cupstestppd \-v '{}' \\;
 
 .fi
 .SH SEE ALSO
--- a/man/filter.man
+++ b/man/filter.man
@@ -44,10 +44,10 @@
 .SH OPTIONS
 Options passed on the command-line typically do not include the default choices
 the printer's PPD file. In addition, some options may be specified in multiple
-ways - "landscape" is a synonym for "orientation-requested=4", "media" is a
+ways \- "landscape" is a synonym for "orientation-requested=4", "media" is a
 synonym for "PageSize", "PageRegion", "InputSlot", and "MediaType", and "sides"
 is a synonym for the various "Duplex" options. Non-raster filters \fBmust\fR
-support both explicit and implicit specification of PPD options - use the
+support both explicit and implicit specification of PPD options \- use the
 ppdMarkDefaults and cupsMarkOptions functions in the CUPS library to use the
 correct mapping, and ppdFindMarkedChoice to get the user-selected choice.
 .LP
@@ -127,7 +127,7 @@
 .TP 5
 STATE: + printer-state-reason [printer-state-reason ...]
 .TP 5
-STATE: - printer-state-reason [printer-state-reason ...]
+STATE: \- printer-state-reason [printer-state-reason ...]
 .br
 Sets, adds, or removes printer-state-reason keywords to the
 current queue. Typically this is used to indicate media, ink, and
--- a/man/ipptool.man
+++ b/man/ipptool.man
@@ -13,22 +13,22 @@
 .\"
 .TH ipptool 1 "CUPS" "9 November 2010" "Apple Inc."
 .SH NAME
-ipptool - perform internet printing protocol requests
+ipptool \- perform internet printing protocol requests
 .SH SYNOPSIS
 .B ipptool
-[ -4 ] [ -6 ] [ -C ] [ -E ] [ -I ] [ -L ] [ -S ] [ -T
+[ \-4 ] [ \-6 ] [ \-C ] [ \-E ] [ \-I ] [ \-L ] [ \-S ] [ \-T
 .I seconds
-] [ -V
+] [ \-V
 .I version
-] [ -X ] [ -c ] [ -d
+] [ \-X ] [ \-c ] [ \-d
 .I name=value
-] [ -f
+] [ \-f
 .I filename
-] [ -i
+] [ \-i
 .I seconds
-] [ -n
+] [ \-n
 .I repeat-count
-] [ -q ] [ -t ] [ -v ]
+] [ \-q ] [ \-t ] [ \-v ]
 .I URI
 .I filename
 [
@@ -39,58 +39,58 @@
 .SH OPTIONS
 The following options are recognized by \fIipptool\fR:
 .TP 5
--4
+\-4
 Specifies that \fIipptool\fR must connect to the printer or server using IPv4.
 .TP 5
--6
+\-6
 Specifies that \fIipptool\fR must connect to the printer or server using IPv6.
 .TP 5
--C
+\-C
 Specifies that requests should be sent using the HTTP/1.1 "Transfer-Encoding: chunked" header, which is required for conformance by all versions of IPP. The default is to use "Transfer-Encoding: chunked" for requests with attached files and "Content-Length:" for requests without attached files.
 .TP 5
--E
+\-E
 Forces TLS encryption when connecting to the server using the HTTP "Upgrade" header.
 .TP 5
--I
+\-I
 Specifies that \fIipptool\fR will continue past errors.
 .TP 5
--L
+\-L
 Specifies that requests should be sent using the HTTP/1.0 "Content-Length:" header, which is required for conformance by all versions of IPP. The default is to use "Transfer-Encoding: chunked" for requests with attached files and "Content-Length:" for requests without attached files.
 .TP 5
--S
+\-S
 Forces (dedicated) SSL encryption when connecting to the server.
 .TP 5
--T seconds
+\-T seconds
 Specifies a timeout for IPP requests in seconds.
 .TP 5
--V version
+\-V version
 Specifies the default IPP version to use: 1.0, 1.1, 2.0, 2.1, or 2.2. If not specified, version 1.1 is used.
 .TP 5
--X
+\-X
 Specifies that XML (Apple plist) output is desired instead of the plain text report. This option is incompatible with the \fI-i\fR (interval) and \fI-n\fR (repeat-count) options.
 .TP 5
--c
+\-c
 Specifies that CSV (comma-separated values) output is desired instead of the plain text output.
 .TP 5
--d name=value
+\-d name=value
 Defines the named variable.
 .TP 5
--f filename
+\-f filename
 Defines the default request filename for tests.
 .TP 5
--i seconds
+\-i seconds
 Specifies that the (last) file should be repeated at the specified interval. This option is incompatible with the \fI-X\fR (XML plist output) option.
 .TP 5
--l
+\-l
 Specifies that plain text output is desired.
 .TP 5
--n repeat-count
+\-n repeat-count
 Specifies that the (last) file should be repeated the specified number of times. This option is incompatible with the \fI-X\fR (XML plist output) option.
 .TP 5
--t
+\-t
 Specifies that CUPS test report output is desired instead of the plain text output.
 .TP 5
--v
+\-v
 Specifies that all request and response attributes should be output in CUPS test mode (\fI-t\fR). This is the default for XML output.
 .SH COMPATIBILITY
 The \fIipptool\fR program is unique to CUPS.
@@ -102,7 +102,7 @@
 .LP
 Send email notifications to "user@example.com" when "myprinter" changes:
 .nf
-    ipptool -d recipient=mailto:user@example.com \
+    ipptool \-d recipient=mailto:user@example.com \
         ipp://localhost/printers/myprinter create-printer-subscription.test
 .fi
 .SH STANDARD FILES
--- a/man/ipptoolfile.man
+++ b/man/ipptoolfile.man
@@ -72,7 +72,7 @@
 .TP 5
 DEFINE variable-name value
 Defines the named variable to the given value. This is equivalent to specifying
-"-d variable-name=value" on the \fIipptool\fR command-line.
+"\-d variable-name=value" on the \fIipptool\fR command-line.
 .TP 5
 DEFINE-DEFAULT variable-name value
 Defines the named variable to the given value if it does not already have a
@@ -118,13 +118,13 @@
 .TP 5
 TRANSFER chunked
 Specifies that tests will, by default, use the HTTP/1.1 "Transfer-Encoding:
-chunked" header. This is the default and is equivalent to specifying "-c" on the
+chunked" header. This is the default and is equivalent to specifying "\-c" on the
 \fIipptool\fR command-line. Support for chunked requests is required for
 conformance with all versions of IPP.
 .TP 5
 TRANSFER length
 Specifies that tests will, by default, use the HTTP/1.0 "Content-Length:"
-header. This is equivalent to specifying "-l" on the \fIipptool\fR command-line.
+header. This is equivalent to specifying "\-l" on the \fIipptool\fR command-line.
 Support for content length requests is required for conformance with all
 versions of IPP.
 .TP 5
@@ -144,7 +144,7 @@
 .TP 5
 ATTR tag attribute-name value(s)
 Adds an attribute to the test request. Values are separated by the comma (",")
-character - escape commas using the "\" character.
+character \- escape commas using the "\" character.
 .TP 5
 ATTR collection attribute-name { MEMBER tag member-name value(s) ... } [ ... { ... } ]
 Adds a collection attribute to the test request. Member attributes follow the
@@ -164,7 +164,7 @@
 .TP 5
 EXPECT !attribute-name
 Specifies that the response must/may/must not include the named attribute.
-Additional requirements can be added as predicates - see the "EXPECT PREDICATES"
+Additional requirements can be added as predicates \- see the "EXPECT PREDICATES"
 section for more information on predicates.
 .TP 5
 FILE filename
@@ -210,7 +210,7 @@
 .TP 5
 STATUS status-code [ predicate ]
 Specifies an expected response status-code value. Additional requirements can be
-added as predicates - see the "STATUS PREDICATES" section for more information
+added as predicates \- see the "STATUS PREDICATES" section for more information
 on predicates.
 .TP 5
 TRANSFER auto
@@ -516,7 +516,7 @@
 .SH VARIABLES
 The \fIipptool\fR program maintains a list of variables that can be used in any
 literal string or attribute value by specifying "$variable-name". Aside from
-variables defined using the "-d" option or "DEFINE" directive, the following
+variables defined using the "\-d" option or "DEFINE" directive, the following
 pre-defined variables are available:
 .TP 5
 $$
@@ -527,7 +527,7 @@
 environment variable is not defined.
 .TP 5
 $filename
-Inserts the filename provided to \fIipptool\fR with the "-f" option.
+Inserts the filename provided to \fIipptool\fR with the "\-f" option.
 .TP 5
 $hostname
 Inserts the hostname from the URI provided to \fIipptool\fR.
--- a/man/lp.man
+++ b/man/lp.man
@@ -14,49 +14,49 @@
 .\"
 .TH lp 1 "CUPS" "16 July 2012" "Apple Inc."
 .SH NAME
-lp - print files
+lp \- print files
 .SH SYNOPSIS
 .B lp
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -c ] [ -d
+] [ \-c ] [ \-d
 .I destination[/instance]
-] [ -h
+] [ \-h
 .I hostname[:port]
-] [ -m ] [ -n
+] [ \-m ] [ \-n
 .I num-copies
-] [ -o
+] [ \-o
 .I option[=value]
-] [ -q
+] [ \-q
 .I priority
-] [ -s ] [ -t
+] [ \-s ] [ \-t
 .I title
-] [ -H
+] [ \-H
 .I handling
-] [ -P
+] [ \-P
 .I page-list
-] [ -- ] [
+] [ \-- ] [
 .I file(s)
 ]
 .br
 .B lp
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -c ] [ -h
+] [ \-c ] [ \-h
 .I hostname[:port]
-] [ -i
+] [ \-i
 .I job-id
-] [ -n
+] [ \-n
 .I num-copies
-] [ -o
+] [ \-o
 .I option[=value]
-] [ -q
+] [ \-q
 .I priority
-] [ -t
+] [ \-t
 .I title
-] [ -H
+] [ \-H
 .I handling
-] [ -P
+] [ \-P
 .I page-list
 ]
 .SH DESCRIPTION
@@ -70,20 +70,20 @@
 .SH OPTIONS
 The following options are recognized by \fIlp\fR:
 .TP 5
---
+\--
 .br
 Marks the end of options; use this to print a file whose name
 begins with a dash (-).
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--U username
+\-U username
 .br
 Specifies the username to use when connecting to the server.
 .TP 5
--c
+\-c
 .br
 This option is provided for backwards-compatibility only. On
 systems that support it, this option forces the print file to be
@@ -91,52 +91,52 @@
 files are always sent to the scheduler via IPP which has the
 same effect.
 .TP 5
--d destination
+\-d destination
 .br
 Prints files to the named printer.
 .TP 5
--h hostname[:port]
+\-h hostname[:port]
 .br
 Chooses an alternate server.
 .TP 5
--i job-id
+\-i job-id
 .br
 Specifies an existing job to modify.
 .TP 5
--m
+\-m
 .br
 Sends an email when the job is completed.
 .TP 5
--n copies
+\-n copies
 .br
 Sets the number of copies to print from 1 to 100.
 .TP 5
--o "name=value [name=value ...]"
+\-o "name=value [name=value ...]"
 .br
 Sets one or more job options.
 .TP 5
--q priority
+\-q priority
 .br
 Sets the job priority from 1 (lowest) to 100 (highest). The
 default priority is 50.
 .TP 5
--s
+\-s
 .br
 Do not report the resulting job IDs (silent mode.)
 .TP 5
--t "name"
+\-t "name"
 .br
 Sets the job name.
 .TP 5
--H hh:mm
+\-H hh:mm
 .TP 5
--H hold
+\-H hold
 .TP 5
--H immediate
+\-H immediate
 .TP 5
--H restart
+\-H restart
 .TP 5
--H resume
+\-H resume
 .br
 Specifies when the job should be printed. A value of \fIimmediate\fR will print
 the file immediately, a value of \fIhold\fR will hold the job indefinitely, and
@@ -145,91 +145,91 @@
 Use a value of \fIrestart\fR with the \fI-i\fR option to restart
 a completed job.
 .TP 5
--P page-list
+\-P page-list
 .br
 Specifies which pages to print in the document. The list can
 contain a list of numbers and ranges (#-#) separated by commas
 (e.g. 1,3-5,16). The page numbers refer to the output pages and
-not the document's original pages - options like "number-up" can
+not the document's original pages \- options like "number-up" can
 affect the numbering of the pages.
 .SH COMMON JOB OPTIONS
 Aside from the printer-specific options reported by the
 \fIlpoptions(1)\fR command, the following generic options are
 available:
 .TP 5
--o media=size
+\-o media=size
 .br
 Sets the page size to \fIsize\fR. Most printers support at least
 the size names "a4", "letter", and "legal".
 .TP 5
--o landscape
+\-o landscape
 .TP 5
--o orientation-requested=4
+\-o orientation-requested=4
 .br
 Prints the job in landscape (rotated 90 degrees).
 .TP 5
--o sides=one-sided
+\-o sides=one-sided
 .TP 5
--o sides=two-sided-long-edge
+\-o sides=two-sided-long-edge
 .TP 5
--o sides=two-sided-short-edge
+\-o sides=two-sided-short-edge
 .br
 Prints on one or two sides of the paper. The value
 "two-sided-long-edge" is normally used when printing portrait
 (unrotated) pages, while "two-sided-short-edge" is used for
 landscape pages.
 .TP 5
--o fit-to-page
+\-o fit-to-page
 .br
 Scales the print file to fit on the page.
 .TP 5
--o number-up=2
+\-o number-up=2
 .TP 5
--o number-up=4
+\-o number-up=4
 .TP 5
--o number-up=6
+\-o number-up=6
 .TP 5
--o number-up=9
+\-o number-up=9
 .TP 5
--o number-up=16
+\-o number-up=16
 .br
 Prints multiple document pages on each output page.
 .TP 5
--o cpi=N
+\-o cpi=N
 .br
 Sets the number of characters per inch to use when printing a
 text file. The default is 10.
 .TP 5
--o lpi=N
+\-o lpi=N
 .br
 Sets the number of lines per inch to use when printing a text
 file. The default is 6.
 .TP 5
--o page-bottom=N
+\-o page-bottom=N
 .TP 5
--o page-left=N
+\-o page-left=N
 .TP 5
--o page-right=N
+\-o page-right=N
 .TP 5
--o page-top=N
+\-o page-top=N
 .br
 Sets the page margins when printing text files. The values are in
-points - there are 72 points to the inch.
+points \- there are 72 points to the inch.
 .SH EXAMPLES
 Print a double-sided legal document to a printer called "foo":
 .nf
-    lp -d foo -o media=legal -o sides=two-sided-long-edge filename
+    lp \-d foo \-o media=legal \-o sides=two-sided-long-edge filename
 .fi
 .LP
 Print an image across 4 pages:
 .nf
-    lp -d bar -o scaling=200 filename
+    lp \-d bar \-o scaling=200 filename
 .fi
 .LP
 Print a text file with 12 characters per inch, 8 lines per inch, and
 a 1 inch left margin:
 .nf
-    lp -d bar -o cpi=12 -o lpi=8 -o page-left=72 filename
+    lp \-d bar \-o cpi=12 \-o lpi=8 \-o page-left=72 filename
 .fi
 .SH COMPATIBILITY
 Unlike the System V printing system, CUPS allows printer names to
--- a/man/lpadmin.man
+++ b/man/lpadmin.man
@@ -17,31 +17,31 @@
 lpadmin \- configure cups printers and classes
 .SH SYNOPSIS
 .B lpadmin
-[ -E ] [-U
+[ \-E ] [\-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] -d
+] \-d
 .I destination
 .br
 .B lpadmin
-[ -E ] [-U
+[ \-E ] [\-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] -p
+] \-p
 .I destination
-[ -R
+[ \-R
 .I name-default
 ]
 .I option(s)
 .br
 .B lpadmin
-[ -E ] [-U
+[ \-E ] [\-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] -x
+] \-x
 .I destination
 .SH DESCRIPTION
 \fIlpadmin\fR configures printer and class queues provided by
@@ -69,75 +69,75 @@
 The following options are recognized when configuring a printer
 queue:
 .TP 5
--c class
+\-c class
 .br
 Adds the named \fIprinter\fR to \fIclass\fR.  If \fIclass\fR does
 not exist it is created automatically.
 .TP 5
--i interface
+\-i interface
 .br
 Sets a System V style interface script for the printer. This
 option cannot be specified with the \fI-P\fR option (PPD file)
 and is intended for providing support for legacy printer drivers.
 .TP 5
--m model
+\-m model
 .br
 Sets a standard System V interface script or PPD file for the printer from the
 \fImodel\fR directory or using one of the driver interfaces. Use the \fI-m\fR
 option with the \fIlpinfo(8)\fR command to get a list of supported models.
 .TP 5
--o cupsIPPSupplies=true
+\-o cupsIPPSupplies=true
 .TP 5
--o cupsIPPSupplies=false
+\-o cupsIPPSupplies=false
 .br
 Specifies whether IPP supply level values should be reported.
 .TP 5
--o cupsSNMPSupplies=true
+\-o cupsSNMPSupplies=true
 .TP 5
--o cupsSNMPSupplies=false
+\-o cupsSNMPSupplies=false
 .br
 Specifies whether SNMP supply level (RFC 3805) values should be reported.
 .TP 5
--o job-k-limit=value
+\-o job-k-limit=value
 .br
 Sets the kilobyte limit for per-user quotas. The value is an
 integer number of kilobytes; one kilobyte is 1024 bytes.
 .TP 5
--o job-page-limit=value
+\-o job-page-limit=value
 .br
 Sets the page limit for per-user quotas. The value is the integer
 number of pages that can be printed; double-sided pages are
 counted as two pages.
 .TP 5
--o job-quota-period=value
+\-o job-quota-period=value
 .br
 Sets the accounting period for per-user quotas. The value is an
 integer number of seconds; 86,400 seconds are in one day.
 .TP 5
--o job-sheets-default=banner
+\-o job-sheets-default=banner
 .TP 5
--o job-sheets-default=banner,banner
+\-o job-sheets-default=banner,banner
 .br
 Sets the default banner page(s) to use for print jobs.
 .TP 5
--o name=value
+\-o name=value
 .br
 Sets a PPD option for the printer. PPD options can be listed using the \fI-l\fR
 option with the \fIlpoptions(1)\fR command.
 .TP 5
--o name-default=value
+\-o name-default=value
 .br
 Sets a default server-side option for the destination. Any print-time
-option can be defaulted, e.g. "-o cpi-default=17" to set the default
+option can be defaulted, e.g. "\-o cpi-default=17" to set the default
 "cpi" option value to 17.
 .TP 5
--o port-monitor=name
+\-o port-monitor=name
 .br
 Sets the binary communications program to use when printing,
 "none", "bcp", or "tbcp". The default program is "none". The
 specified port monitor must be listed in the printer's PPD file.
 .TP 5
--o printer-error-policy=name
+\-o printer-error-policy=name
 .br
 Sets the error policy to be used when the printer backend is
 unable to send the job to the printer. The name must be one of
@@ -145,7 +145,7 @@
 error policy is "stop-printer" for printers and "retry-current-job" for
 classes.
 .TP 5
--o printer-is-shared=true/false
+\-o printer-is-shared=true/false
 .br
 Sets the destination to shared/published or unshared/unpublished.
 Shared/published destinations are publicly announced by the server
@@ -153,53 +153,53 @@
 \fBcupsd.conf\fR, while unshared/unpublished destinations are not
 announced. The default value is "true".
 .TP 5
--o printer-op-policy=name
+\-o printer-op-policy=name
 .br
 Sets the IPP operation policy associated with the destination. The
 name must be defined in the \fBcupsd.conf\fR in a Policy section.
 The default operation policy is "default".
 .TP 5
--R name-default
+\-R name-default
 .br
 Deletes the named option from \fIprinter\fR.
 .TP 5
--r class
+\-r class
 .br
 Removes the named \fIprinter\fR from \fIclass\fR.  If the
 resulting class becomes empty it is removed.
 .TP 5
--u allow:user,user,@group
+\-u allow:user,user,@group
 .TP 5
--u deny:user,user,@group
+\-u deny:user,user,@group
 .TP 5
--u allow:all
+\-u allow:all
 .TP 5
--u deny:none
+\-u deny:none
 .br
 Sets user-level access control on a destination. Names starting with
 "@" are interpreted as UNIX groups. The latter two forms turn
 user-level access control off.
 .TP 5
--v "device-uri"
+\-v "device-uri"
 .br
 Sets the \fIdevice-uri\fR attribute of the printer queue. Use the \fI-v\fR
 option with the \fIlpinfo(8)\fR command to get a list of supported device URIs
 and schemes.
 .TP 5
--D "info"
+\-D "info"
 .br
 Provides a textual description of the destination.
 .TP 5
--E
+\-E
 .br
 Enables the destination and accepts jobs; this is the same as running the
 \fIcupsaccept(8)\fR and \fIcupsenable(8)\fR programs on the destination.
 .TP 5
--L "location"
+\-L "location"
 .br
 Provides a textual location of the destination.
 .TP 5
--P ppd-file
+\-P ppd-file
 .br
 Specifies a PostScript Printer Description file to use with the
 printer. If specified, this option overrides the \fI-i\fR option
--- a/man/lpinfo.man
+++ b/man/lpinfo.man
@@ -17,36 +17,36 @@
 lpinfo \- show available devices or drivers
 .SH SYNOPSIS
 .B lpinfo
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ -l ] [ --device-id
+] [ \-l ] [ \--device-id
 .I device-id-string
-] [ --exclude-schemes
+] [ \--exclude-schemes
 .I scheme-list
-] [ --include-schemes
+] [ \--include-schemes
 .I scheme-list
-] [ --language
+] [ \--language
 .I locale
-] [ --make-and-model
+] [ \--make-and-model
 .I name
-] [ --product
+] [ \--product
 .I name
-] -m
+] \-m
 .br
 .B lpinfo
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ -l ] [ --exclude-schemes
+] [ \-l ] [ \--exclude-schemes
 .I scheme-list
-] [ --include-schemes
+] [ \--include-schemes
 .I scheme-list
-] [ --timeout
+] [ \--timeout
 .I seconds
-] -v
+] \-v
 .SH DESCRIPTION
 \fIlpinfo\fR lists the available devices or drivers known to the
 CUPS server. The first form (\fI-m\fR) lists the available
@@ -55,51 +55,51 @@
 .SH OPTIONS
 \fIlpinfo\fR accepts the following options:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--U username
+\-U username
 .br
 Sets the username to use when connecting to the server.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Selects an alternate server.
 .TP 5
--l
+\-l
 .br
 Shows a "long" listing of devices or drivers.
 .TP 5
---device-id device-id-string
+\--device-id device-id-string
 .br
 Specifies the IEEE-1284 device ID to match when listing drivers with the
 \fI-m\fR option.
 .TP 5
---exclude-schemes scheme-list
+\--exclude-schemes scheme-list
 .br
 Specifies a comma-separated list of device or PPD schemes that should be
 excluded from the results. Static PPD files use the "file" scheme.
 .TP 5
---include-schemes scheme-list
+\--include-schemes scheme-list
 .br
 Specifies a comma-separated list of device or PPD schemes that should be
 included in the results. Static PPD files use the "file" scheme.
 .TP 5
---language locale
+\--language locale
 .br
 Specifies the language to match when listing drivers with the \fI-m\fR option.
 .TP 5
---make-and-model name
+\--make-and-model name
 .br
 Specifies the make and model to match when listing drivers with the \fI-m\fR
 option.
 .TP 5
---product name
+\--product name
 .br
 Specifies the product to match when listing drivers with the \fI-m\fR option.
 .TP 5
---timeout seconds
+\--timeout seconds
 .br
 Specifies the timeout when listing devices with the \fI-v\fR option.
 .SH COMPATIBILITY
--- a/man/lpmove.man
+++ b/man/lpmove.man
@@ -17,17 +17,17 @@
 lpmove \- move a job or all jobs to a new destination
 .SH SYNOPSIS
 .B lpmove
-[ -E ] [ -h
+[ \-E ] [ \-h
 .I server[:port]
-] [ -U
+] [ \-U
 .I username
 ]
 .I job destination
 .br
 .B lpmove
-[ -E ] [ -h
+[ \-E ] [ \-h
 .I server[:port]
-] [ -U
+] [ \-U
 .I username
 ]
 .I source destination
@@ -44,15 +44,15 @@
 .SH OPTIONS
 The \fIlpmove\fR command supports the following options:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--U username
+\-U username
 .br
 Specifies an alternate username.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Specifies an alternate server.
 .SH SEE ALSO
--- a/man/lpoptions.man.in
+++ b/man/lpoptions.man.in
@@ -17,90 +17,90 @@
 lpoptions \- display or set printer options and defaults
 .SH SYNOPSIS
 .B lpoptions
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] -d
+] \-d
 .I destination[/instance]
-[ -o
+[ \-o
 .I option[=value]
-] ... [ -o
+] ... [ \-o
 .I option[=value]
 ]
 .br
 .B lpoptions
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ -p
+] [ \-p
 .I destination[/instance]
-] -l
+] \-l
 .br
 .B lpoptions
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ -o
+] [ \-o
 .I option[=value]
-] ... [ -o
+] ... [ \-o
 .I option[=value]
-] [ -p
+] [ \-p
 .I destination[/instance]
-] -r
+] \-r
 .I option
 .br
 .B lpoptions
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] -x
+] \-x
 .I destination[/instance]
 .SH DESCRIPTION
 \fIlpoptions\fR displays or sets printer options and defaults.
 \fIlpoptions\fR shows the default printer options when run with no
 arguments. Other options include:
 .TP 5
--E
+\-E
 .br
 Enables encryption when communicating with the CUPS server.
 .TP 5
--U username
+\-U username
 .br
 Uses an alternate username.
 .TP 5
--d destination[/instance]
+\-d destination[/instance]
 .br
 Sets the user default printer to \fIdestination\fR. If \fIinstance\fR
 is supplied then that particular instance is used. This option
 overrides the system default printer for the current user.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Uses an alternate server.
 .TP 5
--l
+\-l
 .br
 Lists the printer specific options and their current settings.
 .TP 5
--o option[=value]
+\-o option[=value]
 .br
 Specifies a new option for the named destination.
 .TP 5
--p destination[/instance]
+\-p destination[/instance]
 .br
 Sets the destination and instance, if specified, for any options
 that follow. If the named instance does not exist then it is
 created.
 .TP 5
--r option
+\-r option
 .br
 Removes the specified option for the named destination.
 .TP 5
--x destination[/instance]
+\-x destination[/instance]
 .br
 Removes the options for the named destination and instance, if
 specified. If the named instance does not exist then this does
@@ -119,10 +119,10 @@
 .SH COMPATIBILITY
 The \fIlpoptions\fR command is unique to CUPS.
 .SH FILES
-~/.cups/lpoptions - user defaults and instances created by non-root
+~/.cups/lpoptions \- user defaults and instances created by non-root
 users.
 .br
-@CUPS_SERVERROOT@/lpoptions - system-wide defaults and instances
+@CUPS_SERVERROOT@/lpoptions \- system-wide defaults and instances
 created by the root user.
 .SH SEE ALSO
 \fIcancel(1)\fR, \fIlp(1)\fR, \fIlpadmin(8)\fR, \fIlpr(1)\fR,
--- a/man/lppasswd.man
+++ b/man/lppasswd.man
@@ -22,13 +22,13 @@
 ]
 .br
 .B lppasswd
--a [ -g
+\-a [ \-g
 .I groupname
 ]
 .I username
 .br
 .B lppasswd
--x
+\-x
 .I username
 .SH DESCRIPTION
 \fIlppasswd\fR adds, changes, or deletes passwords in the CUPS
@@ -41,7 +41,7 @@
 .SH OPTIONS
 \fIlppasswd\fR supports the following options:
 .TP 5
--g groupname
+\-g groupname
 .br
 Specifies a group other than the default system group.
 .SH SECURITY ISSUES
--- a/man/lpr.man
+++ b/man/lpr.man
@@ -17,17 +17,17 @@
 lpr \- print files
 .SH SYNOPSIS
 .B lpr
-[ -E ] [ -H
+[ \-E ] [ \-H
 .I server[:port]
-] [ -U
+] [ \-U
 .I username
-] [ -P
+] [ \-P
 .I destination[/instance]
-] [ -#
+] [ \-#
 .I num-copies
-[ -h ] [ -l ] [ -m ] [ -o
+[ \-h ] [ \-l ] [ \-m ] [ \-o
 .I option[=value]
-] [ -p] [ -q ] [ -r ] [ -C/J/T
+] [ \-p] [ \-q ] [ \-r ] [ \-C/J/T
 .I title
 ] [
 .I file(s)
@@ -45,65 +45,65 @@
 .SH OPTIONS
 The following options are recognized by \fIlpr\fR:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--H server[:port]
+\-H server[:port]
 .br
 Specifies an alternate server.
 .TP 5
--C "name"
+\-C "name"
 .TP 5
--J "name"
+\-J "name"
 .TP 5
--T "name"
+\-T "name"
 .br
 Sets the job name.
 .TP 5
--P destination[/instance]
+\-P destination[/instance]
 .br
 Prints files to the named printer.
 .TP 5
--U username
+\-U username
 .br
 Specifies an alternate username.
 .TP 5
--# \fIcopies\fR
+\-# \fIcopies\fR
 .br
 Sets the number of copies to print from 1 to 100.
 .TP 5
--h
+\-h
 .br
-Disables banner printing. This option is equivalent to "-o
+Disables banner printing. This option is equivalent to "\-o
 job-sheets=none".
 .TP 5
--l
+\-l
 .br
 Specifies that the print file is already formatted for the
 destination and should be sent without filtering. This option is
-equivalent to "-o raw".
+equivalent to "\-o raw".
 .TP 5
--m
+\-m
 .br
 Send an email on job completion.
 .TP 5
--o option[=value]
+\-o option[=value]
 .br
 Sets a job option.
 .TP 5
--p
+\-p
 .br
 Specifies that the print file should be formatted with a shaded
 header with the date, time, job name, and page number. This
-option is equivalent to "-o prettyprint" and is only useful when
+option is equivalent to "\-o prettyprint" and is only useful when
 printing text files.
 .TP 5
--q
+\-q
 .br
 Hold job for printing.
 .TP 5
--r
+\-r
 .br
 Specifies that the named print files should be deleted after
 printing them.
--- a/man/lprm.man
+++ b/man/lprm.man
@@ -17,13 +17,13 @@
 lprm \- cancel print jobs
 .SH SYNOPSIS
 .B lprm
-[ -E ] [ -U
+[ \-E ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I server[:port]
-] [ -P
+] [ \-P
 .I destination[/instance]
-] [ - ] [
+] [ \- ] [
 .I job ID(s)
 ]
 .SH DESCRIPTION
@@ -35,19 +35,19 @@
 .SH OPTIONS
 The \fIlprm\fR command supports the following options:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--P destination[/instance]
+\-P destination[/instance]
 .br
 Specifies the destination printer or class.
 .TP 5
--U username
+\-U username
 .br
 Specifies an alternate username.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Specifies an alternate server.
 .SH COMPATIBILITY
--- a/man/lpstat.man
+++ b/man/lpstat.man
@@ -17,23 +17,23 @@
 lpstat \- print cups status information
 .SH SYNOPSIS
 .B lpstat
-[ -E ] [ -H ] [ -U
+[ \-E ] [ \-H ] [ \-U
 .I username
-] [ -h
+] [ \-h
 .I hostname[:port]
-] [ -l ] [ -W
+] [ \-l ] [ \-W
 .I which-jobs
-] [ -a [
+] [ \-a [
 .I destination(s)
-] ] [ -c [
+] ] [ \-c [
 .I class(es)
-] ] [ -d ] [ -o [
+] ] [ \-d ] [ \-o [
 .I destination(s)
-] ] [ -p [
+] ] [ \-p [
 .I printer(s)
-] ] [ -r ] [ -R ] [ -s ] [ -t ] [ -u [
+] ] [ \-r ] [ \-R ] [ \-s ] [ \-t ] [ \-u [
 .I user(s)
-] ] [ -v [
+] ] [ \-v [
 .I printer(s)
 ] ]
 .SH DESCRIPTION
@@ -43,23 +43,23 @@
 .SH OPTIONS
 The \fIlpstat\fR command supports the following options:
 .TP 5
--E
+\-E
 .br
 Forces encryption when connecting to the server.
 .TP 5
--H
+\-H
 .br
 Shows the server hostname and port.
 .TP 5
--R
+\-R
 .br
 Shows the ranking of print jobs.
 .TP 5
--U username
+\-U username
 .br
 Specifies an alternate username.
 .TP 5
--W which-jobs
+\-W which-jobs
 .br
 Specifies which jobs to show, \fIcompleted\fR or
 \fInot-completed\fR (the default). This option \fImust\fR appear
@@ -67,60 +67,60 @@
 the default (not-completed) value will be used in the request to
 the scheduler.
 .TP 5
--a [printer(s)]
+\-a [printer(s)]
 .br
 Shows the accepting state of printer queues. If no printers are
 specified then all printers are listed.
 .TP 5
--c [class(es)]
+\-c [class(es)]
 .br
 Shows the printer classes and the printers that belong to them.
 If no classes are specified then all classes are listed.
 .TP 5
--d
+\-d
 .br
 Shows the current default destination.
 .TP 5
--h server[:port]
+\-h server[:port]
 .br
 Specifies an alternate server.
 .TP 5
--l
+\-l
 .br
 Shows a long listing of printers, classes, or jobs.
 .TP 5
--o [destination(s)]
+\-o [destination(s)]
 .br
 Shows the jobs queue on the specified destinations. If no destinations are
 specified all jobs are shown.
 .TP 5
--p [printer(s)]
+\-p [printer(s)]
 .br
 Shows the printers and whether or not they are enabled for printing. If
 no printers are specified then all printers are listed.
 .TP 5
--r
+\-r
 .br
 Shows whether the CUPS server is running.
 .TP 5
--s
+\-s
 .br
 Shows a status summary, including the default destination, a
 list of classes and their member printers, and a list of printers and
-their associated devices. This is equivalent to using the "-d", "-c",
-and "-v" options.
+their associated devices. This is equivalent to using the "\-d", "\-c",
+and "\-v" options.
 .TP 5
--t
+\-t
 .br
-Shows all status information. This is equivalent to using the "-r",
-"-d", "-c", "-v", "-a", "-p", and "-o" options.
+Shows all status information. This is equivalent to using the "\-r",
+"\-d", "\-c", "\-v", "\-a", "\-p", and "\-o" options.
 .TP 5
--u [user(s)]
+\-u [user(s)]
 .br
 Shows a list of print jobs queued by the specified users. If no users
 are specified, lists the jobs queued by the current user.
 .TP 5
--v [printer(s)]
+\-v [printer(s)]
 .br
 Shows the printers and what device they are attached to. If no printers
 are specified then all printers are listed.
@@ -129,9 +129,9 @@
 contain any printable character except SPACE, TAB, "/", and "#".
 Also, printer and class names are \fInot\fR case-sensitive.
 .LP
-The "-h", "-E", "-U", and "-W" options are unique to CUPS.
+The "\-h", "\-E", "\-U", and "\-W" options are unique to CUPS.
 .LP
-The Solaris "-f", "-P", and "-S" options are silently ignored.
+The Solaris "\-f", "\-P", and "\-S" options are silently ignored.
 .SH SEE ALSO
 \fIcancel(1)\fR, \fIlp(1)\fR,
 .br
--- a/man/mime.types.man
+++ b/man/mime.types.man
@@ -40,7 +40,7 @@
 logical AND and "," or whitespace for a logical OR, and negated using
 "!".
 .SH RULES
-Rules take two forms - a filename extension by itself and functions with test
+Rules take two forms \- a filename extension by itself and functions with test
 values inside parenthesis. The following functions are available:
 .TP 5
 match("pattern")
--- a/man/ppdc.man
+++ b/man/ppdc.man
@@ -17,17 +17,17 @@
 ppdc \- cups ppd compiler
 .SH SYNOPSIS
 .B ppdc
-[ -D
+[ \-D
 .I name[=value]
-] [ -I
+] [ \-I
 .I include-directory
-] [ -c
+] [ \-c
 .I message-catalog
-] [ -d
+] [ \-d
 .I output-directory
-] [ -l
+] [ \-l
 .I language(s)
-] [-m] [-t] [ -v ] [ -z ] [ --cr ] [ --crlf ] [ --lf ]
+] [\-m] [\-t] [ \-v ] [ \-z ] [ \-\-cr ] [ \-\-crlf ] [ \-\-lf ]
 .I source-file
 .SH DESCRIPTION
 \fIppdc\fR compiles PPDC source files into one or more PPD
@@ -68,7 +68,7 @@
 The default is to generate uncompressed PPD files.
 .PP
 The \fI--cr\fR, \fI--crlf\fR, and \fI--lf\fR options specify the
-line ending to use - carriage return, carriage return and line feed,
+line ending to use \- carriage return, carriage return and line feed,
 or line feed. The default is to use the line feed character alone.
 .SH SEE ALSO
 ppdhtml(1), ppdi(1), ppdmerge(1), ppdpo(1), ppdcfile(5)
--- a/man/ppdhtml.man
+++ b/man/ppdhtml.man
@@ -17,9 +17,9 @@
 ppdhtml \- cups html summary generator
 .SH SYNOPSIS
 .B ppdhtml
-[ -D
+[ \-D
 .I name[=value]
-] [ -I
+] [ \-I
 .I include-directory
 ]
 .I source-file
--- a/man/ppdpo.man
+++ b/man/ppdpo.man
@@ -17,11 +17,11 @@
 ppdpo \- ppd message catalog generator
 .SH SYNOPSIS
 .B ppdpo
-[ -D
+[ \-D
 .I name[=value]
-] [ -I
+] [ \-I
 .I include-directory
-] [ -o
+] [ \-o
 .I output-file
 ]
 .I source-file
